### PR TITLE
Replace rocket hashes with Ruby 1.9 ones

### PR DESCRIPTION
**Pull request by tnir**
### What was the end-user problem that led to this PR?

Rocket hashes are used in Ruby code.

### What was your diagnosis of the problem?

They can be replaced with new ones.

### What is your fix for the problem, implemented in this PR?

Replace rocket hashes in Ruby code with Ruby 1.9 ones.

### Why did you choose this fix out of the possible options?

Simplified code would give a better contributor experience.

No other cops are not enabled in this PR as per the title.

Signed-off-by: Takuya Noguchi [takninnovationresearch@gmail.com](https://github.com/sponsors/tnir)

### CODE REVIEW

**Review by deivid-rodriguez — COMMENTED** [review]
Can you enable the rule in `.rubocop.yml` and let RuboCop auto-correct everything?

**Comment by deivid-rodriguez** [comment]
Why only rake tasks?

File: .rubocop.yml [hunk 1]
```diff
@@ -3,3 +3,7 @@ AllCops:
 
 Layout/TrailingWhitespace:
   Enabled: true
+
+Style/HashSyntax:
+  EnforcedStyle: ruby19
+  EnforcedShorthandSyntax: never
```
Comment (deivid-rodriguez) [inline .rubocop.yml:9]: I didn't know about this new shorthand syntax feature. It looks cool, why disable its use?

File: Rakefile [hunk 2]
```diff
@@ -1,11 +1,11 @@
 Dir.glob("lib/tasks/**/*.rake").each { |r| load r }
 
 desc "Build the static site"
-task :build => ["contributors:update", :repo_pages, :man] do
+task build: %w[contributors:update repo_pages man] do
```
Comment (deivid-rodriguez) [inline Rakefile:4]: Can we also enable the cop to enforce `%w[]` for Arrays of Strings?

File: .rubocop.yml (hunk 1 above)
Comment (tnir) [reply .rubocop.yml:9]: Existing codes are all not in shorthand, so I just wanted to minimize the changes.

File: Rakefile (hunk 2 above)
Comment (tnir) [reply Rakefile:4]: We can do it in another PR.

Comment (tnir) [reply Rakefile:4]: Or, I can revert this part to the following in this PR:

```suggestion
task build: ["contributors:update", :repo_pages, :man] do
```

**Comment by tnir** [comment]
> Why only rake tasks?

I did not have a plan to change or consider other codes than Rake tasks.

File: Rakefile (hunk 2 above)
Comment (tnir) [reply Rakefile]: Reverted.

**Comment by tnir** [comment]
Back to you.

**Review by hsbt — APPROVED** [review]

**Comment by hsbt** [comment]
Thanks!

**Comment by deivid-rodriguez** [comment]
Thanks ❤️ !

**Pull request merged**